modules/gdscript/tests/scripts/parser/errors/function_conflicts_constant.gd

```diff
@@ -1,5 +1,4 @@
 const test = 25
 
-
```

Comment:
There is still this line that was removed.

Reply:
Oops! Fixed it.